src/raft/recv_install_snapshot.c

```diff
@@ -262,6 +261,7 @@
 
 enum rpc_state {
 	RPC_INIT,
+	RPC_FILLED,
```

Comment:
[Question for reviewer]: I introduced this new state because I wanted to differentiate the state where the rpc exists and the state where it is ready to be sent, meaning it is filled (we have called `rpc_fill`). I tried to make it explicit to avoid handling it as a special case. Also it is needed in order to be able to react to `M_MSG_SENT` (see my other comment).

Reply:
I like the additional state.